scanpipe/pipes/purldb.py

```diff
@@ -117,3 +117,13 @@ def match_resource(sha1_list, timeout=None, api_url=PURLDB_API_URL):
     if response and response.get("count"):
         packages = response["results"]
         return packages
+
+
+def fetch_package(purl, timeout=None, api_url=PURLDB_API_URL):
+    """Fetch package data for the PURL."""
+    payload = {"purl": purl}
+    response = request_get(url=f"{api_url}packages/", payload=payload, timeout=timeout)
```

Comment:
We did not have one before?

Reply:
We didn't have one for getting package using PURL